packages/react-components/react-tabster/src/index.ts

```diff
@@ -57,6 +57,7 @@ export { KEYBORG_FOCUSIN } from 'keyborg';
 
 // WARNING! ATTENTION! Tabster.Types was exported from here by mistake. To avoid breaking changes,
 // we are putting a snapshot of Tabster.Types@6.0.1 and marking the entire export as deprecated.
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Idk, maybe because of this: [#7269](https://github.com/typescript-eslint/typescript-eslint/pull/7269). It's still failing even after dedupe, so I guess it should be expected behaviour 🤔